src/bsp/common/stm32/startup_stm32u575zitxq.c

```diff
@@ -184,6 +184,139 @@ void MDF1_FLT5_IRQHandler         (void) __attribute__ ((weak, alias("Default_Ha
 void CORDIC_IRQHandler            (void) __attribute__ ((weak, alias("Default_Handler")));
 void FMAC_IRQHandler              (void) __attribute__ ((weak, alias("Default_Handler")));
 
+
```

Comment:
Added for future debugging needs. I used this heavily for the RBR resets debugging.